### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dizzyn**
Removed link. It's better to not link 404 pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ $dom
 ====
 _5k Javascript library for selecting, styling, traversing and animating DOM elements._
 
-$dom was coded by Keith Clark and was once available at http://www.keithclark.co.uk/$dom/.
+$dom was coded by Keith Clark and was once available at ````http://www.keithclark.co.uk/$dom/```` (no longer active).
```
Comment (julienw) [inline README.md]: just curious: why 4 backticks and not only 1 ?

Comment (dizzyn) [inline README.md]: by mistake :) I'm not using MD daily

Comment (julienw) [inline README.md]: I was surprised that the 4 backticks actually work :D

**Comment by julienw** [comment]
landed in https://github.com/julienw/dollardom/commit/43fb83444d14bd3de82ab8d6f09161e60c3dcdcf

thanks !

**Pull request closed without merging**